deploy/hostpathprovisioner_cr.yaml

```diff
@@ -6,7 +6,16 @@ spec:
   imagePullPolicy: Always
   storagePools:
     - name: "local"
+      storageClass:
+        name: "local"
```

Comment:
this yaml seems kind of confusing to me, is this referring to the "local" storage class?  Why not use storageClassName in the pvcTemplate?

Reply:
You make a good point. Let me fix that.